_i18n/ar/resources/user-guides/weblate/remote_node_gui.po

```diff
@@ -17,158 +17,129 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 
 #. type: Plain text
-#: _i18n/en/resources/user-guides/remote_node_gui.md:2
+#: ../_i18n/en/resources/user-guides/remote_node_gui.md:2
```

Comment:
Are these changes of path expected?

Reply:
i believe so, it is what happens when the po-update.sh script is used. 
 the "../" is in relation to the po-update.sh file https://github.com/monero-project/monero-site/blob/master/po/po-update.sh 

an example of this here:
https://github.com/monero-project/monero-site/blob/5c15000bc3d3980a13d2d1a73d8c2364d19967cc/_i18n/ar/resources/moneropedia/weblate/account.po#L26